pom.xml

```diff
@@ -25,11 +25,11 @@ under the License.
   <parent>
     <groupId>org.jenkins-ci.plugins</groupId>
     <artifactId>plugin</artifactId>
-    <version>2.17</version>
+    <version>2.32</version>
   </parent>
 
   <artifactId>saml</artifactId>
-  <version>0.15-SNAPSHOT</version>
+  <version>1.00-SNAPSHOT</version>
```

Comment:
@kuisathaverat Do you mean `1.0.0-SNAPSHOT` or `1.0-SNAPSHOT`? `1.00` sounds like a weird version naming.